app/src/main/java/io/gnosis/safe/ui/safe/add/AddSafeNameFragment.kt

```diff
@@ -60,11 +66,11 @@ class AddSafeNameFragment : BaseViewBindingFragment<FragmentAddSafeNameBinding>(
                         when (action) {
                             is CloseScreen -> {
                                 requireActivity().hideSoftKeyboard()
-                                findNavController().popBackStack(R.id.addSafeFragment, true)
+                                findNavController().popBackStack(R.id.selectChainFragment, true)
                             }
                             is ImportOwner -> {
                                 requireActivity().hideSoftKeyboard()
-                                findNavController().navigate(AddSafeNameFragmentDirections.actionAddSafeNameFragmentToAddSafeOwnerFragment(binding.addSafeNameEntry.text.toString(), navArgs.newAddress))
+                                findNavController().navigate(AddSafeNameFragmentDirections.actionAddSafeNameFragmentToAddSafeOwnerFragment(binding.addSafeNameEntry.text.toString(), navArgs.newAddress, selectedChain))
```

Comment:
owner should be chain agnostic. I think we don't need chain data there as owners are added not for a specific chain